uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/filters/due.js

```diff
@@ -25,7 +25,12 @@ define([
       switch(type){
         case 'overdue':
           for (i; i < inputArrlen; i++) {
-            if (new Date(inputArr[i].activity_date_time).setHours(0, 0, 0, 0) < today) {
+            itemDueDate = null;
```

Comment:
yes its important for the comparison : 

`if (itemDueDate < today && itemDueDate !== null) {`